components/concept/storage/src/main/java/mozilla/components/concept/storage/CreditCardsStorage.kt

```diff
@@ -0,0 +1,202 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.storage
+
+/**
+ * An interface which defines read/write methods for credit card and address data.
+ */
+@SuppressWarnings("TooManyFunctions")
+interface CreditCardsStorage {
```

Comment:
As annoying as it is, I think a `CreditCardsAddressesStorage` is a better name, since this store owns two data types.

Reply:
Same goes for the file name.